app/views/collections/_actions.html.erb

```diff
@@ -14,7 +14,7 @@
 	%>
   </li>
   <li class="list-group-item">
-    <%= link_to_if can?(:edit, current_object),
+    <%= link_to_if can?(:create, METSFolder.new(collection_id: current_object.id)),
```

Comment:
This is a bit awkward in that you shouldn't have to create a new object in order to authorize.  I think it might be better to authorize an action on current_object -- something like `:ingest_metadata` or `:update_metadata`.